templates/db-connect.php

```diff
@@ -1,8 +1,8 @@
 <?php 
 
 $servername = "localhost";
-$username = "siv2r";
```

Comment:
@siv2r rest all seems to be fine, but should this be removed too?

Reply:
@riasenpai yeah, It is better to remove it.